sdks/go/pkg/beam/model/pipeline_v1/beam_runner_api.pb.go

```diff
@@ -6242,11 +6289,11 @@ var fileDescriptor_cf57597c3a9659a9 = []byte{
 
 // Reference imports to suppress errors if they are not otherwise used.
 var _ context.Context
-var _ grpc.ClientConn
+var _ grpc.ClientConnInterface
```

Comment:
@lostluck I'm not sure why this line and L6296 were changed. Do you have any idea?

Reply:
It's fixed by following https://github.com/apache/beam/blob/master/sdks/go/pkg/beam/model/PROTOBUF.md#generated-go-code-fails-to-build